packages/radio-player/src/search-bar/assets/img/clear-results.ts

```diff
@@ -0,0 +1,7 @@
+import { html } from 'lit-html';
```

Comment:
Why are these SVG objects converted to lit-html templates rather than just referenced via `<img src` or `background-image`? There don't appear to be any optional arguments to them to allow us to change attributes before using them.